.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
+    "standard.vscode-standard"
+  ]
+}
```

Comment:
never played around with commiting vscode settings before? what exactly does having this extensions.json/recomendations block do?

Reply:
For those who use vscode - it will recommend extensions that can be used to make things better for development. These extensions are specific to the project / repo.

Reply:
Awesome! didn't know you could do that! but ill be making use of that on future projects!